src/main/java/org/opendevstack/provision/services/BaseServiceAdapter.java

```diff
@@ -87,12 +88,16 @@ public RestClientCall httpDelete() {
   private RestClientCall authenticatedCall(HttpVerb verb) {
     RestClientCall call = notAuthenticatedCall(verb);
 
-    if (useTechnicalUser) {
-      return call.basicAuthenticated(new CredentialsInfo(userName, userPassword));
+    try {
+      if (useTechnicalUser) {
+        return call.basicAuthenticated(new CredentialsInfo(userName, userPassword));
+      }
+      CredentialsInfo credentialsInfo =
```

Comment:
the tech user thing should never be null if in use - and they are configured hard..
I woudl move the try to l95